docs/source/etl/merge.rst

```diff
@@ -0,0 +1,92 @@
+.. _merge-datasets:
+
+========================
+Merging Variant Datasets
+========================
+
+.. invisible-code-block:: python
```

Comment:
FYI, there is only a single colon after invisible code blocks. We get warnings with double colons.